message_sender/factory.py

```diff
@@ -361,8 +364,22 @@ def return_response(self, response):
             response.raise_for_status()
         except requests_exceptions.HTTPError as exc:
             resp = exc.response.text
+
             if not ("1006" in resp and "unknown contact" in resp):
                 raise exc
+
+                if "410" in resp:
+                    if (
+                        self.time_period == 0
+                        and self.time_period < self.max_time_period
+                    ):
+                        self.time_period += 7
+                        raise exc  # or send message?
+                    elif self.time_period < self.max_time_period:
+                        self.time_period += 7
+                elif self.time_period >= self.max_time_period:
+                    self.time_period = 0
```

Comment:
We don't need to be handling this here. WhatsApp doesn't return this error immediately (else the HTTP request would take 7 days), it returns it to us in a separate webhook back to us.

Reply:
ohh I see, okay